config/config.go

```diff
@@ -0,0 +1,45 @@
+package config
```

Comment:
After reviewing in more detail - it's pretty clear to me that `config.go` actually just belongs in the `server/` package - it's configuration options for the server commands nothing else (not client, nor keys, etc.). There is no import problems - it's only referenced in  `server` and `gaia` right now - so after the move, it will only be imported by `gaia`